src/client/components/HelpText/HelpText.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './HelpText.css';
+
+export default function HelpText(props) {
+  const klassName = props.klass || '';
+  let txt = props.text;
+
+  if (!txt) {
+    txt =
+      'If you already do not have a slack id then please make' +
+      ' ' +
+      'one as it is mandatory to have a slack id to connect with' +
+      ' ' +
+      'mentors and other students.';
+  }
```

Comment:
You should not have this hard-coded, the text comes from the outside. You are just providing a style.